src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
+        Truck truck = new Truck();
+        excavator.doWork();
+        bulldozer.doWork();
+        truck.doWork();
+        excavator.stopWork();
+        bulldozer.stopWork();
+        truck.stopWork();
```

Comment:
You should call methods doWork() and stopWork() in a loop. Try to add all machines to array